tests/app/template_email_files/test_rest.py

```diff
@@ -87,6 +89,50 @@ def test_create_fails_if_template_does_not_exist(client, sample_service):
         )
 
 
+def test_template_update_bumps_new_file_template_version(client, sample_service, sample_email_template):
+    file_one_data = {
```

Comment:
it feels like we should have `sample_email_file` fixture, to make these tests terser and less repeatable.

Reply:
created the fixture, needs applying to other tests